applications/PfemFluidDynamicsApplication/python_scripts/remesh_fluid_domains_process.py

```diff
@@ -164,21 +166,24 @@ def ExecuteInitializeSolutionStep(self):
         currentTime=self.main_model_part.ProcessInfo[KratosMultiphysics.TIME]
         currentStep=self.main_model_part.ProcessInfo[KratosMultiphysics.STEP]                
 
-        if(currentStep > 1):
+        if currentStep >= 2 and self.fileTotalVolume is None:
+            self.fileTotalVolume = open("totalVolumeBeforeMeshing.txt",'w')
```

Comment:
usually  FileName.post.csv is a better extension to know that is is a results file and a type of excel file